dist/api_data.py

```diff
@@ -1255,6 +1255,11 @@ def __ge__(self, other):
             whether pre-fetch is enabled for all sessions by default''',
             type='boolean'),
         ]),
+    Config('preserve_prepared', 'false', r'''
```

Comment:
Can we indicate there that this is a feature in develop?

Reply:
The intention of this flag is that it can be used for the attached storage also in the future; due to this reason, it is not dependent on the disaggregated storage flag. 

Adding more information about it may not be needed.